### PR TITLE
[Edit] GIF added to cursor.md

### PR DESCRIPTION
**Pull request by DanWSDev**
### Description
Added a GIF to show how the code in the example.

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/cursor/cursor.md [hunk 1]
```diff
@@ -54,3 +54,4 @@ In the example below, a `pointer` hand will appear whenever the cursor hovers ov
 ```html
 <p style="cursor: pointer;">The cursor is pointing at this paragraph!</p>
 ```
+![cursor-example](https://github.com/Codecademy/docs/assets/148011207/572b61ea-d6c3-42be-b87e-30a670f261c0)
```
Comment (caupolicandiaz) [inline content/css/concepts/cursor/cursor.md:57]: this is the wrong path it should be -
https://raw.githubusercontent.com/Codecademy/docs/main/media/cursor-example.gif

Please add the file to the media folder and use a filename that explains the content, similar to the path above

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@DanWSDev I've added a couple of comments. Please update, and tag me when you've made the changes.

File: content/css/concepts/cursor/cursor.md (hunk 1 above)
Comment (DanWSDev) [reply content/css/concepts/cursor/cursor.md]: @caupolicandiaz Thank you for the correction I believe I have done this now let me know if anything else needs to be changed.

**Review by caupolicandiaz — APPROVED** [review]
@DanWSDev This looks good. The one minor edit I would suggest is that there should be an empty line between the code block and the link. This is ready for a second review

**Comment by DanWSDev** [comment]
> @DanWSDev This looks good. The one minor edit I would suggest is that there should be an empty line between the code block and the link. This is ready for a second review

Made the change you suggested

**Review by SSwiniarski — APPROVED** [review]
@DanWSDev Looks good.

**Pull request merged**